projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -1,26 +1,51 @@
 <div class="go-table-container" *ngIf="showTable">
-  <table class="go-table" *ngIf="hasData(); else noDataTable">
-    <thead>
-      <tr>
-        <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
-          <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
-          <ng-template #columnHeaderOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
-          </ng-template>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr *ngFor="let item of localTableConfig.tableData" class="go-table__tr">
-        <td *ngFor="let col of columns" class="go-table__td" width="{{ col.width || 'auto' }}">
-          <ng-container *ngIf="!col.goTableCell; else columnCellOutlet">{{ col.getFieldData(item) }}</ng-container>
-          <ng-template #columnCellOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableCell; context: { $implicit: item }"></ng-container>
-          </ng-template>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+  <div class="go-table__table">
+    <table class="go-table" *ngIf="hasData(); else noDataTable">
+      <thead>
+        <tr>
+          <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
```

Comment:
What if we called this class `go-table__heading`?

Reply:
In the past I've used a simple svg to establish the same effect as the `[ngClass]="columnClasses(col.field)"`. We could potentially even wrap that svg in a button to get bonus accessibility points. I don't know that we need to make this change now, but it could alternatively look something like this:
```html
<svg height="20" viewBox="0 0 24 24">
  <desc>up and down arrows for controlling table column sort direction</desc>
  <polygon points='9 3 5 6.99 8 6.99 8 14 10 14 10 6.99 13 6.99 9 3'></polygon>
  <polygon points='15 21 19 17.01 16 17.01 16 10 14 10 14 17.01 11 17.01 15 21'></polygon>
</svg>
```
The benefit being more control over the state of the UI and some additional accessibility.

Reply:
- I thought about `go-table__heading`, but I felt that we _could_ possibly have a heading that's not a `th` in which case, we could reserve that class for such a case. I also thought that `th` was very specific. These styles should only be applied to a `th`.

- This is a pretty good idea I think we can make use of in the redesign of the table.

Reply:
Seems like a valid assumption, but maybe one that we don't need to make yet?